### ISSUE TITLE
Specify loading the external dictionary

### ISSUE THREAD

**Issue by Yoric** [binast/container-format-spec#4]
We don't want the VM to need to download stuff, so this has to be done externally, i.e. by DOM.

**Comment by dominiccooney** [comment]
In addition to specifying actual fetches, etc. we probably need a preload type for these resources so browsers can start the dictionary and first BinAST fetch in parallel.

**Comment by sendilkumarn** [comment]
I think adding a `preload` is the best way to download the dictionary 
But does that mean that each `script` tag (if there are multiple) would require this `attribute` and make `vm` intelligent enough to handle this?

**Comment by Yoric** [comment]
A few precisions after discussing with Annevk:

- we don't want same origin policy for dictionaries, as this would prevent us from putting dictionaries on a cdn;
- we want CORS, as all new features pretty much require CORS.

**Comment by Yoric** [comment]
I agree that a `preload` would be nice, but I believe that this should be a HTTP header, which could be returned either with the webpage itself or with the .binjs. This will make it easier for proxies to inject such preloads.

@RReverser, if you're around, what do you think about that?
